libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
No, that would break backwards compatibility. Right now you can write:

```
define X {
  if ( continue ) { X } else { nullProcess }
}
```

Which cannot be statically expanded during parsing (the expansion is infinite), it must be done at runtime.